--- origsrc/krb5-1.12.1/src/lib/apputils/net-server.c	2014-05-18 14:44:56.633171400 -0500
+++ src/krb5-1.12.1/src/lib/apputils/net-server.c	2014-05-16 01:41:18.000000000 -0500
@@ -1380,7 +1380,7 @@ send_to_from(int s, void *buf, size_t le
     msg.msg_controllen = 0;
 
     switch (from->sa_family) {
-#if defined(IP_PKTINFO)
+#if defined(IP_PKTINFO) && !defined(__CYGWIN__)
     case AF_INET:
         if (fromlen != sizeof(struct sockaddr_in))
             goto use_sendto;
